src/App.js

```diff
@@ -0,0 +1,25 @@
+import React, {useState} from 'react';
+import { Switch, Route } from 'react-router-dom';
+import { NavBar, Footer } from './layout';
+import {Homepage, Repopage} from './pages'
+
+
+function App() {
+    return (
+        <>
+        <NavBar />
+        <Switch>
+            <Route exact path ="/">
+                <Homepage />
+            </Route>
+            <Route path ="/Repopage">
+                <Repopage />
+            </Route>
+        </Switch>
+        <Footer />
+        </>
+       
```

Comment:
Clever to use NavBar and Footer here so you do not have to reference them again on each page